lichbot.lic

```diff
@@ -122,10 +123,87 @@ class Lichbot
     fput "whisper #{character} Supported commands are:"
     fput "whisper #{character}   (help) - this list"
     fput "whisper #{character}   (inv|inventory) - view the items I have available"
+    fput "whisper #{character}   (bal|balance) - view your current bot balance"
+    fput "whisper #{character}   (with|withdraw) [amount] - withdraw [amount] from your bot balance."
     fput "whisper #{character}   (give|giveme) [item] - take an item from me"
     fput "whisper #{character}   (buffs|spells) - view the buffs I can cast"
     fput "whisper #{character}   (buff|cast) [abbreviation] - receive [abbreviation] buff from me"
-    fput "whisper #{character} I will also accept and stow any item you trade me"
+    fput "whisper #{character} I will also accept and stow any item you trade me, or accept coins via TIPs to hold for you."
+  end
+
+  def balance(character)
+    @last_action = Time.now
+    unless has_bot_account(character)
```

Comment:
Sure, makes sense to create new accounts/empty set.  And yes, that's one of the problems with GIVE -- it's not impossible to work around, of course, but not sure how much value I'd get out of going that way. 

TIPs are more transactional, which is helpful when managing balances. Plus, if 3 people give you coins at the same time (before balances/current money on-hand is evaluated) it could quickly screw things up -- or add a lot of extra logic to properly handle these cases.